### ISSUE TITLE
Does this bot still work ?

### ISSUE THREAD

**Issue by Androc** [SupremoRTD/LoyaltyBot#7]
Hello,

I am looking for a loyalty bot and found yours.
But it was not touched since 2 years.

Does it still work ?
Do you know an other good Loyalty bot if yours is not maintain anymore ?

Thanks in advance.

**Comment by Comedyrussell** [comment]
The new loyalty bot I use is ankhbotR2

**Comment by Androc** [comment]
Thanks for your answer. Is it this one http://marcinswierzowski.com/AnkhBot.html ?

Why do you use it ? Only for loyalty ? Do you have tested others ?

**Comment by Comedyrussell** [comment]
Yes I find that bot very helpful and cool to use it has a lot of features
On Jul 16, 2015 9:23 AM, "Androc" <notifications@github.com> wrote:
>
> Thanks for your answer. Is it this one
http://marcinswierzowski.com/AnkhBot.html ?

**Comment by Androc** [comment]
Ok thanks. I am going to have a look. Do you know if it is still maintained ?

**Comment by Comedyrussell** [comment]
Im pretty sure it is maintained by a twitch streamer I would have to find
out their name I can tell you in about 10 - 20 mins
On Jul 16, 2015 9:27 AM, "Androc" <notifications@github.com> wrote:
>
> Ok thanks. I am going to have a look. Do you know if it is still
maintained ?

**Comment by Androc** [comment]
"Maintained" = "In active development".

**Comment by Comedyrussell** [comment]
yes i understand, and yes the bot is maintained

**Comment by Androc** [comment]
Thanks. I am trying it. I am interested in the currency system to reward my viewers. It is updated every hour ?

**Comment by Comedyrussell** [comment]
the currency? or the bot?

**Comment by Androc** [comment]
The currency system in the bot. I don't find a parameter to change the frequency of the update of points. So I suppose it happens only each hour
